frontend/src/SettingsPage/SettingsPage.jsx

```diff
@@ -122,6 +154,7 @@ function SettingsPage(props) {
                         placeholder="Enter last name"
                         value={lastName}
                         onChange={(event) => setLastName(event.target.value)}
+                        onKeyPress={(event) => !/^[A-Za-z]+$/.test(event.key) && event.preventDefault(true)}
```

Comment:
> this will conflict #2494 should accept special characters for first name and last name,

okay, working on it.

Reply:
Hi @gsmithun4, after I run `git pull` for fetching the latest commit and run the docker, it will resulting an error


![image](https://user-images.githubusercontent.com/5271698/173603945-7861b1c5-5e2b-418e-b542-211ce05344d1.png)
![image](https://user-images.githubusercontent.com/5271698/173604036-87a9bd62-6730-41d4-b55b-44bfc4c6203a.png)
![image](https://user-images.githubusercontent.com/5271698/173604155-728969c6-2c07-460d-954d-d5694c74f250.png)

Reply:
Hi @mazkaaa, can you please try `npm install --prefix frontend` and restart

Reply:
It’s s working now. thank you!